guides/source/action_view_overview.md

```diff
@@ -402,1179 +401,7 @@ This will add `app/views/direct` to the end of the lookup paths.
-
-#### sanitize_css(style)
-
-Sanitizes a block of CSS code.
-
-#### strip_links(html)
-Strips all link tags from text leaving just the link text.
-
-```ruby
-strip_links('<a href="https://rubyonrails.org">Ruby on Rails</a>')
-# => Ruby on Rails
-```
-
-```ruby
-strip_links('emails to <a href="mailto:me@email.com">me@email.com</a>.')
-# => emails to me@email.com.
-```
-
-```ruby
-strip_links('Blog: <a href="http://myblog.com/">Visit</a>.')
-# => Blog: Visit.
-```
-
-#### strip_tags(html)
-
-Strips all HTML tags from the html, including comments.
-This functionality is powered by the rails-html-sanitizer gem.
-
-```ruby
-strip_tags("Strip <i>these</i> tags!")
-# => Strip these tags!
-```
-
-```ruby
-strip_tags("<b>Bold</b> no more!  <a href='more.html'>See more</a>")
-# => Bold no more!  See more
-```
-
-NB: The output may still contain unescaped '<', '>', '&' characters and confuse browsers.
-
-### UrlHelper
-
-Provides methods to make links and get URLs that depend on the routing subsystem.
-
-#### url_for
-
-Returns the URL for the set of `options` provided.
-
-##### Examples
-
-```ruby
-url_for @profile
-# => /profiles/1
-
-url_for [ @hotel, @booking, page: 2, line: 3 ]
-# => /hotels/1/bookings/1?line=3&page=2
-```
-
-#### link_to
-
-Links to a URL derived from `url_for` under the hood. Primarily used to
-create RESTful resource links, which for this example, boils down to
-when passing models to `link_to`.
-
-**Examples**
-
-```ruby
-link_to "Profile", @profile
-# => <a href="/profiles/1">Profile</a>
-```
-
-You can use a block as well if your link target can't fit in the name parameter. ERB example:
-
-```html+erb
-<%= link_to @profile do %>
-  <strong><%= @profile.name %></strong> -- <span>Check it out!</span>
-<% end %>
-```
-
-would output:
-
-```html
-<a href="/profiles/1">
-  <strong>David</strong> -- <span>Check it out!</span>
-</a>
-```
-
-See [the API Doc for more info](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to)
-
-#### button_to
-
-Generates a form that submits to the passed URL. The form has a submit button
-with the value of the `name`.
-
-##### Examples
-
-```html+erb
-<%= button_to "Sign in", sign_in_path %>
-```
-
-would roughly output something like:
-
-```html
-<form method="post" action="/sessions" class="button_to">
-  <input type="submit" value="Sign in" />
-</form>
-```
-
-See [the API Doc for more info](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-button_to)
-
-### CsrfHelper
-
-Returns meta tags "csrf-param" and "csrf-token" with the name of the cross-site
-request forgery protection parameter and token, respectively.
-
-```html
-<%= csrf_meta_tags %>
-```
-
-NOTE: Regular forms generate hidden fields so they do not use these tags. More
-details can be found in the [Rails Security Guide](security.html#cross-site-request-forgery-csrf).
+This overview has been moved to a separate guide: [Action View Helpers Guide](action_view_helpers.html).
```

Comment:
What do you think about changing this "moved" notice into more of a segue at the end of this guide?

Something to the effect of: "Rails provides many helper methods to use with Action View.  For an overview of these helpers, see the [Action View Helpers Guide](action_view_helpers.html)."

The section title should also probably be changed to reflect its (lack of) content, e.g. just "Action View Helpers".

Reply:
@jonathanhefner I've changed this section. I've included some examples and the section now links to both the Action View Helpers and Action View Form Helpers guides.

Reply:
<img width="609" alt="image" src="https://user-images.githubusercontent.com/28561/85059897-ad643580-b1a4-11ea-88bc-b663c035d440.png">

Reply:
That looks great!